concent_api/common/logging.py

```diff
@@ -258,6 +261,47 @@ def log(
         raise TypeError('Unexpected logging level')
 
 
+def log_received_force_payment(
+    logger: Logger,
+    force_payment_message: ForcePayment,
+):
+    log(logger, f"ForcePayment received with {len(force_payment_message.subtask_results_accepted_list)} nested SubtaskResultsAccepted.")
+    for subtask_results_accepted in force_payment_message.subtask_results_accepted_list:
+        log(logger, f"ForcePayment message contains SubtaskResultsAccepted with SUBTASK_ID: {subtask_results_accepted.subtask_id}.")
+
+
+def log_different_keys_addresses_and_wrong_requestor_signature(
+    logger: Logger,
+    subtask_results_accepted_list: List[SubtaskResultsAccepted],
+):
+    for subtask_results_accepted in subtask_results_accepted_list:
+        log(logger, f"Keys and addresses are not unique in {subtask_results_accepted.__class__.__name__} in "
+                    f"subtask_results_accepted_list and they are not signed by the same requestor", subtask_id=subtask_results_accepted.subtask_id)
+
+
+def log_duplicate_subtask_results_accepted(
+    logger: Logger,
+    subtask_results_accepted_list: List[SubtaskResultsAccepted],
+):
+    log(logger, f"SubtaskResultsAccepted messages are duplicated in subtask_results_accepted_list {subtask_results_accepted_list}")
+
+
+def log_lack_of_unsettled_task_found(
+    logger: Logger,
+    subtask_results_accepted_list: List[SubtaskResultsAccepted],
+):
+    for subtask_results_accepted in subtask_results_accepted_list:
+        log(logger, f"Lack of unsettled payments.", subtask_id=subtask_results_accepted.subtask_id)
+
+
+def log_payment_time_exceeded(
+    logger: Logger,
+    subtask_results_accepted_list: List[SubtaskResultsAccepted],
+):
+    for subtask_results_accepted in subtask_results_accepted_list:
+        log(logger, f"Time for payment exceeded cut off time. Payment rejected for SUBTASK_ID: {subtask_results_accepted.subtask_id}")
```

Comment:
please pass `subtask_id` as `log` parameter. This function unifies style of all logs and makes easier to search messages. Please change to:
```python
log(logger, f"ForcePayment message contains SubtaskResultsAccepted", subtask_id=subtask_results_accepted.subtask_id)
```

Reply:
Already outdated because this log will be added in https://github.com/golemfactory/concent/pull/1015#discussion_r241343589